oshmem/mca/sshmem/verbs/sshmem_verbs_component.c

```diff
@@ -100,6 +101,11 @@ verbs_runtime_query(mca_base_module_t **module,
     *priority = 0;
     *module = NULL;
 
+    /* If fork support is requested, try to enable it */
+    if (OSHMEM_SUCCESS != (rc = opal_common_verbs_fork_test())) {
+        return OSHMEM_ERROR;
```

Comment:
There is a strange disparity between this and oob ud: in oob ud, if common_verbs_fork_test() fails, you opal_output_verbose().  But here, you just return an error.

Should probably be consistent between all places that call common_verbs_fork_test().